src/OrchardCore.Themes/TheAgencyTheme/TheAgencyTheme.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore.Modules\OrchardCore.Lucene\OrchardCore.Lucene.csproj" />
```

Comment:
Nope. If there is an interface you need from this module, then there should be an abstractions project.

Reply:
I agree. I kind just wanted to split the PR in two parts and passed less time thinking about how fixing the Search views but an abstraction would make sense since the search results could come from Lucene or eventually ElasticSearch.